### DISCUSSION TITLE
Can not create a new organization

### DISCUSSION THREAD

**Discussion by Gabriel-p** [community/community#36806] [Other Feature Feedback, Questions, & Ideas]
A couple of weeks ago I created an organization and then deleted it (I was testing something). Now I want to create it again but I get the error:

`Organization name 'xxx' is unavailable.`

The name is not taken (in which case the error would be `The name 'xxx' is already taken.`), it just says `unavailable`. What does it mean and how can I re-create that organization?

Thanks

**Comment by ossanna16** [comment]
Thank you for asking a good question @Gabriel-p, I'm glad you found our community 🙂
